### DISCUSSION TITLE
【常见问题】如何安装 Frodium 整合包？

### DISCUSSION THREAD

**Discussion by ScandiumMascot** [Scandium-Studio/Frodium-Modpack#17] [Q&A 你问我答] [answered]
如题

**Comment by WForst-Breeze** [comment, marked as answer]
1. 您可以在 _Code_ 页面右侧找到 _Releases_ 栏，点击标有 _Latest_ 的版本；
2. 在打开的页面中，单击 _Assets_ 下的 _Source code (zip)_；
3. 将下载好的压缩包文件拖入启动器（eg：PCL2、HMCL）中，开始自动安装。
